app/assets/json/translations/react/en.json

```diff
@@ -841,6 +841,7 @@
   "listings.Mobility impairments.title": "Mobility Impairments",
   "listings.Mobility/Hearing/Vision impairments.title": "Mobility, Hearing and/or Vision Impairments",
   "listings.Vision impairments.title": "Vision Impairments",
+  "listings.HCBS Units.title": "In-home Support for a Disability",
```

Comment:
note: this entry is not in alphabetical order

Reply:
Good catch, they will be auto-sorted when I do the Phrase translation process, but I prefer to keep them ordered beforehand to minimize diffs.